langchain/agents/tests/json.test.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from "@jest/globals";
+import { JsonListKeysTool, JsonSpec, JsonGetValueTool } from "../tools/json";
+
+test("JsonListKeysTool", async () => {
+  const jsonSpec = new JsonSpec({
+    foo: "bar",
+    baz: { test: { foo: [1, 2, 3] } },
```

Comment:
What about lists of dictionaries, do we support that?

Reply:
Yeah, we should. Will add test case for this